app/views/search/_sitetour_modal.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="modal fade sitetour" id="sitetourModal" style="font-family: FontAwesome;" tabindex="-1" role="dialog">
```

Comment:
Yeah, I'm cool with using FA in this modal... we just want to make sure that we are setting our CSS in CSS files rather than inline with our HTML.